extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -280,12 +280,14 @@ class AmpBridPlayer extends AMP.BaseElement {
    */
   getConsentData_() {
     const consentPolicy = super.getConsentPolicy() || 'default';
```

Comment:
```suggestion
    const consentPolicy = this.getConsentPolicy() || 'default';
```

Reply:
Sorry for not seeing this is noted in your previous request @alanorozco . This is now changed.